packages/ketcher-core/src/application/editor/shared/constants.ts

```diff
@@ -34,3 +34,5 @@ export const selectionKeys = [
   'simpleObjects',
   'texts',
 ] as const;
+
+export const defaultBondThickness = 1;
```

Comment:
Can we please set this to `2` and refer to it in here: https://github.com/epam/ketcher/blob/master/packages/ketcher-react/src/script/ui/data/schema/options-schema.ts#L224

Reply:
or probably even create a separate constant for this and refer to it in both places

Reply:
done